core/domain/feedback_jobs_continuous_test.py

```diff
@@ -147,16 +147,17 @@ def test_multiple_threads_multiple_exp(self):
             ModifiedFeedbackAnalyticsAggregator.start_computation()
             self.assertEqual(self.count_jobs_in_taskqueue(), 1)
             self.process_and_flush_pending_tasks()
+            feedback_analytics = (
```

Comment:
@wxyxinyu I'm not sure if @seanlip meant that I should keep get_thread_analytics . But your comment seems more reasonable in this case. Thus, I've decided to keep both get_thread_analytics and get_thread_analytics_multi.

I'll write more tests for get_thread_analytics_multi